client/app/templates/login-error.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="cell">
+  <div class="grid-container">
+    <div class="grid-x grid-padding-x grid-padding-y align-middle">
+      <div class="cell large-6 large-offset-3 text-center">
+        <h2>Oops!</h2>
+
+        {{#if this.contactNotAssigned}}
+          <p data-test-applicant-error-message="contact-not-assigned">
+            You are not currently assigned to a DCP project. Please email&nbsp;
+            <a href="mailto:ZAP_feedback_DL@planning.nyc.gov">ZAP_feedback_DL@planning.nyc.gov</a>&nbsp;
+            for further assistance.
+          </p>
+        {{else}}
+          <p>
+            Something went wrong.
+          </p>
+        {{/if}}
+
+        {{#each @model.errors as |error idx|}}
+          {{#if error.detail}}
+            <code class="callout display-block text-left red-muted" data-test-error-message={{idx}}>{{error.detail}}</code>
+          {{else}}
+            {{#each-in error.response as |key value|}}
+              <code class="callout display-block text-left red-muted" data-test-error-response={{concat key idx}}>
+                {{key}}: {{value}}
+              </code>
+            {{/each-in}}
+            <code class="callout display-block text-left red-muted">
+              status: {{error.status}}
+            </code>
+          {{/if}}
+        {{/each}}
```

Comment:
What do you think about this:

What if we put all of it into the loop here? For example, you could say: 

```handlebars
  {{#if (eq error.code 'NO_CONTACT_FOUND')}}
    {{!-- markup here --!}}
  {{else}}
    {{!-- generic error templates here --!}}
  {{/if}}
```

thoughts? it just seems like that computed is a little too much code for what it's actaully doing.

Reply:
Would this go on lines 7-17, or 23-30 ?

If in 23-30, is the idea to only display the user-friendly message, instead of both the user-friendly message and the API message?

Reply:
Nope, I'm saying remove `this.contactNotAssigned` completely, including the computed, but use the markup there and put the code i have written in 23-30. 

The idea is to put the relevant markup inside the loop that's already there to keep that markup in a place where a loop is already happening.